### PR TITLE
LPS-170694 Change Facet Template Classes

### PR DESCRIPTION
**Pull request by BryanEngler**

### CODE REVIEW

**Comment by BryanEngler** [comment]
ci:test:sf

**Comment by BryanEngler** [comment]
ci:test:relevant

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by BryanEngler** [comment]
ci:test:relevant

**Comment by BryanEngler** [comment]
ci:stop

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by BryanEngler** [comment]
ci:test:relevant

File: modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/upgrade/registry/SearchWebUpgradeStepRegistrator.java [hunk 1]
```diff
@@ -36,6 +39,13 @@ public void register(Registry registry) {
 			new UpgradePortletPreferences());
 
 		registry.register("1.0.0", "2.0.0", new SearchPortletUpgradeProcess());
+
+		registry.register(
+			"1.0.0", "2.1.0",
```
Comment (lipusz) [inline modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/upgrade/registry/SearchWebUpgradeStepRegistrator.java:44]: Changing to `"2.0.0", "2.1.0",` seems to be fixing the problem of the upgrade step not being executed. :bulb:

Comment (lipusz) [reply modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/upgrade/registry/SearchWebUpgradeStepRegistrator.java:44]: https://liferay.slack.com/archives/G982WB4TH/p1670869027888299?thread_ts=1670519527.771389&cid=G982WB4TH

**Comment by joshuacords** [comment]
continued here: https://github.com/liferay-search/liferay-portal/pull/698

**Comment by BryanEngler** [comment]
ci:test:sf

**Comment by BryanEngler** [comment]
ci:test:relevant

**Comment by joshchong** [comment]
ci:test:bundle

**Pull request closed without merging**